mslib/mscolab/forms.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.mscolab.forms.py
+    ~~~~~~~~~~~~~~~~~~~~
```

Comment:
in ReST heading definition line needs same length as the word to become the heading